src/main/resources/db/changelog/changes/07-insert-and-assign-roles.yaml

```
@@ -0,0 +1,27 @@
+databaseChangeLog:
+  - changeSet:
+      id: insert_roles
+      author: nikname2303
+      changes:
+        - sql:
+            sql: INSERT INTO roles (name) VALUES ('ADMIN'), ('USER')
+
+  - changeSet:
+      id: assign_role_to_admin
+      author: nikname2303
+      changes:
+        - sql:
+            sql: INSERT INTO user_roles (user_id, role_id)
+              SELECT id, (SELECT id FROM roles WHERE name = 'ADMIN')
+              FROM users
+              WHERE email = 'admin@example.com';
+
+  - changeSet:
+      id: assign_role_to_user
+      author: nikname2303
+      changes:
+        - sql:
+            sql: INSERT INTO user_roles (user_id, role_id)
+              SELECT id, (SELECT id FROM roles WHERE name = 'USER')
+              FROM users
+              WHERE email = 'john@example.com';
```

```suggestion
             - insert:
            tableName: user_roles
            columns:
              - column:
                  name: user_id
                  valueComputed: "(SELECT id FROM users WHERE email = 'john@example.com')"
              - column:
                  name: role_id
                  valueComputed: "(SELECT id FROM roles WHERE role = 'USER')"
```
I believe this way is  more clear

---

But does the same thing so i'ts up to u)